packages/business-rules/src/schemas/householder-appeal/appeal-schema/decision-date/decision-date-validation.js

```diff
@@ -0,0 +1,7 @@
+const dateValidation = require('../../generic-validators/date-validation');
+
+const decisionDateValidation = () => {
+	return dateValidation();
```

Comment:
in these validation modules that return a generic, do you need to create a separate file? can't you just call the dateValidation() (or whatever) function in packages/business-rules/src/schemas/householder-appeal/appeal-schema/index.js